README.rst

```diff
@@ -113,7 +116,7 @@ To use this with your project you need to follow these steps:
         print(res.json())
 
 #.  Check stdout for the printable output, and navigate to ``/admin/log_outgoing_requests/outgoingrequestslog/`` to see 
-    the saved log records
+    the saved log records. The settings for saving logs can by overridden under ``/admin/log_outgoing_requests/outgoingrequestslogconfig/``.
```

Comment:
I'd prefer a bit more human friendly descriptions here, something like:

> Navigate to the **Admin > log_outgoing_requests > Configuration** and ...

You can also tweak the display name of the app in https://github.com/maykinmedia/django-log-outgoing-requests/blob/main/log_outgoing_requests/apps.py#L4 by specifying `verbose_name = _("Outgoing requests logs")` so it's a bit more human readable.

You don't have to worry about the django-admin-index integration here.